omero/developers/python3-migration.rst

```diff
@@ -0,0 +1,108 @@
+Migration from OMERO 5.5 (Python 2) to OMERO 5.6 (Python 3)
+===========================================================
+
+This page serves as a collection of tips the OME team used to
```

Comment:
OME team used tips to get through the migration ? 

Maybe better: ``...recommendations, which come up when OME team went through the migration to Python 3.``